[2024-11-14T02:18:12.586+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-14T02:18:13.389+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stockprices_v2_decorator.return_last_90d_price scheduled__2024-11-14T02:00:00+00:00 [queued]>
[2024-11-14T02:18:16.303+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stockprices_v2_decorator.return_last_90d_price scheduled__2024-11-14T02:00:00+00:00 [queued]>
[2024-11-14T02:18:16.439+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-14T02:18:17.017+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): return_last_90d_price> on 2024-11-14 02:00:00+00:00
[2024-11-14T02:18:17.285+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1848) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-14T02:18:17.481+0000] {standard_task_runner.py:63} INFO - Started process 1875 to run task
[2024-11-14T02:18:17.439+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stockprices_v2_decorator', 'return_last_90d_price', 'scheduled__2024-11-14T02:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/stockprices_v2_decorator(3).py', '--cfg-path', '/tmp/tmpnssistgc']
[2024-11-14T02:18:17.559+0000] {standard_task_runner.py:91} INFO - Job 70: Subtask return_last_90d_price
[2024-11-14T02:18:22.236+0000] {task_command.py:426} INFO - Running <TaskInstance: stockprices_v2_decorator.return_last_90d_price scheduled__2024-11-14T02:00:00+00:00 [running]> on host 3627cdc70e56
[2024-11-14T02:18:40.528+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stockprices_v2_decorator' AIRFLOW_CTX_TASK_ID='return_last_90d_price' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-14T02:00:00+00:00'
[2024-11-14T02:18:41.139+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-14T02:18:42.314+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-14T02:18:42.598+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/stockprices_v2_decorator(3).py", line 49, in return_last_90d_price
    vantage_api_key = Variable.get('VANTAGE_API_KEY')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable VANTAGE_API_KEY does not exist'
[2024-11-14T02:18:43.171+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=stockprices_v2_decorator, task_id=return_last_90d_price, run_id=scheduled__2024-11-14T02:00:00+00:00, execution_date=20241114T020000, start_date=20241114T021813, end_date=20241114T021843
[2024-11-14T02:18:43.709+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 70 for task return_last_90d_price ('Variable VANTAGE_API_KEY does not exist'; 1875)
[2024-11-14T02:18:44.607+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-11-14T02:18:45.283+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T02:18:45.381+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
